packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -313,6 +317,95 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   // stylelint-enable plugin/selector-bem-pattern
 }
 
+.mdc-textfield--with-leading-icon {
+  .mdc-textfield__leading-icon:not([tabindex]),
+  .mdc-textfield__leading-icon[tabindex="-1"] {
+    pointer-events: none;
+    cursor: default;
+  }
+
+  .mdc-textfield__trailing-icon:not([tabindex]),
+  .mdc-textfield__trailing-icon[tabindex="-1"] {
+    pointer-events: none;
+    cursor: default;
+  }
+
+  .mdc-textfield__input {
```

Comment:
This needs to be mirrored into the trailing styles; currently, if you enter a lot of text, it will flow right into the trailing icon.